public/app/core/components/UserEdit/UserEdit.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { PureComponent } from 'react';
+import { hot } from 'react-hot-loader';
+import { connect } from 'react-redux';
+import { StoreState } from 'app/types';
+import Page from 'app/core/components/Page/Page';
+import UserProfile from './UserProfile';
+import UserSessions from './UserSessions';
+import NewUserPassword from './Admin/NewUserPassword';
+import UserTeams from './UserTeams';
+import SharedPreferences from 'app/core/components/SharedPreferences/SharedPreferences';
+import { loadUser } from './state/actions';
+import { OrgUser } from 'app/types';
+import { getRouteParamsId } from 'app/core/selectors/location';
+
+export interface Props {
+  adminMode?: boolean;
+  uid?: number;
```

Comment:
uid is a type of id we use in dashboards & data sources (unique id) as type of guid. So better to use name userId here for clarity